features/support/pages/app.rb

```diff
@@ -24,6 +31,20 @@ def logged_in_as?(user: nil, email: nil)
     end
   end
 
+  def load(page)
+    loaded_pages[page] ||= PAGES[page].new
+    loaded_pages[page].load
+    self.current_page = loaded_pages[page]
```

Comment:
I'm really not sure if caching is good or bad here. I'm kind of tempted to not cache since there doesn't seem to be a clear reason *to* do so; and I would not be surprised if there was pollution caused by caching.